fxa/__main__.py

```diff
@@ -0,0 +1,205 @@
+
+    # Creation args
+    parser.add_argument('--user-salt',
+                        help=('Salt used to calculate the user credentials. '
+                              '(Random by default)'),
+                        dest='fxa_user_salt',
+                        required=False)
+
+    # FxA server configuration
+    parser.add_argument('--account-server',
+                        help='Firefox Account server URL',
+                        dest='account_server_url',
+                        required=False,
+                        default=FXA_API_URL)
+
+    parser.add_argument('--oauth-server',
+                        help='Firefox Account OAuth server URL',
+                        dest='oauth_server_url',
+                        required=False,
+                        default=FXA_OAUTH_URL)
+
+    parser.add_argument('--client-id',
+                        help='Firefox Account OAuth client id.',
+                        dest='client_id',
+                        required=False,
+                        default=DEFAULT_CLIENT_ID)
+
+    parser.add_argument('--scopes',
+                        help='Firefox Account OAuth scopes.',
+                        dest='scopes',
+                        required=False,
+                        default='profile')
+
+    parser.add_argument('--audience',
+                        help='Firefox BrowserID assertion audience.',
+                        dest='audience',
+                        required=False,
+                        default='https://token.services.mozilla.com/')
+
+    parser.add_argument('--duration',
+                        help='Firefox BrowserID assertion duration.',
+                        dest='duration',
+                        required=False,
+                        default='3600')
+
+    parser.add_argument('--prefix',
+                        help='Firefox Account user creation email prefix.',
+                        dest='prefix',
+                        required=False,
+                        default='fxa')
+
+    args = vars(parser.parse_args())
+    create = args['create']
+    auth = args.get('auth')
+    verbose = args['verbose']
+
+    account_server_url = args['account_server_url']
+    oauth_server_url = args['oauth_server_url']
+
+    fd = sys.stdout  # By default write to the standard output
+    out = args.get('output_file')
+    if out:
+        out = os.path.abspath(out)
+        file_path = os.path.dirname(out)
+        if not os.path.exists(file_path):
+            os.makedirs(file_path)
+        fd = open(out, 'w')
+
+    if auth:
+        # Ask for the user password if needed
+        auth = auth.split(':', 1)
+        if len(auth) < 2:
+            email = auth[0]
+            password = getpass.getpass('Please enter a password for %s: '
+                                       % auth[0])
+    elif create:
+        # Create a new user
+        if verbose:
+            print('# Creating the account...', end='', file=sys.stderr)
```

Comment:
Maybe not worth the extra effort, but you could consider using `logging.info` and friends for these messages, and having `--verbose` just set the default logging config when run from the command-line.